graviti/portex/factory.py

```diff
@@ -495,19 +540,19 @@ def __call__(self, **kwargs: Any) -> Fields:
 
 def type_factory_creator(
     decl: Dict[str, Any], imports: Imports
-) -> Union[TypeFactory, DynamicTypeFactory]:
+) -> Union[TypeFactory, DictUnpackTypeFactory]:
     """Check whether the input is dynamic and returns the corresponding type factory.
 
     Arguments:
         decl: A dict which indicates a portex type or a dynamic portex type.
         imports: The :class:`Imports` instance to specify the import scope of the template.
 
     Returns:
-        A ``TypeFactory`` or a ``DynamicTypeFactory`` instance according to the input.
+        A ``TypeFactory`` or a ``DictUnpackTypeFactory`` instance according to the input.
 
     """
-    if decl["type"].startswith("$params."):
```

Comment:
which means the DynamicType is blocked after this commit?
should we block it after implementing all unpack grammars?